cita-network/src/node_manager.rs

```diff
@@ -34,13 +34,15 @@ use std::{
     time::{Duration, Instant},
 };
 
-pub const DEFAULT_MAX_CONNECTS: usize = 4;
+pub const DEFAULT_MAX_CONNECTS: usize = 666;
```

Comment:
666?

Reply:
I like 666.

Reply:
???? Just that? Is it ok for whatevre num?